pkg/builder/cluster/convert/convert.go

```diff
@@ -464,25 +527,29 @@ func ToCRD(pod *corev1.Pod) (*v1alpha1.Build, error) {
 		steps = steps[1:]
 	}
 
-	var scm *v1alpha1.SourceSpec
-	if conv, ok := containerToSourceMap[steps[0].Name]; ok {
-		src, err := conv(steps[0])
-		if err != nil {
-			return nil, err
-		}
-		// The first init container is actually a source step.  Convert
-		// it to our source spec and pop it off the list of steps.
-		scm = src
-		if subPath != "" {
-			scm.SubPath = subPath
+	var scm []*v1alpha1.SourceSpec
+	for _, step := range steps {
+		conv := stepNameConversionToSource(step.Name)
+		if conv != nil {
```

Comment:
conv is not used outside the scope of this if statement, please combine this line with the prev one `if conv := sNCTS(); conv != nil {`